code/_onclick/item_attack.dm

```diff
@@ -115,6 +115,10 @@
 	if(user.mind && user.mind.cm_skills)
 		power = round(power * (1 + 0.3*user.mind.cm_skills.melee_weapons)) //30% bonus per melee level
 
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		power *= H.gloves.force_multiplier
```

Comment:
Events-based.
You have several options:
1) Add a `living` attack modifier variable which gets increased or decreased by adding or removing gloves.
2) Give the melee spec higher melee skill.